### DISCUSSION TITLE
no find Stars

### DISCUSSION THREAD

**Discussion by luohuiiqng** [pelicanmapping/osgearth#2394] [Support Q&A]
After adding the sky, only the sun and moon are observed, no stars are seen.

ref_ptr<SkyNode> ViewerWidget::addSky()
{
    osgEarth::DateTime dateTime(2023, 12, 5, 20);
    ref_ptr<SkyNode> sky_node = SkyNode::create();
    osgEarth::Util::SkyOptions skyOptions;    
    osgEarth::Util::Ephemeris* ephemeris = new osgEarth::Util::Ephemeris;
    sky_node->setMoonVisible(true);
    sky_node->setName("sky");   
    sky_node->setDateTime(dateTime);
    sky_node->setLighting(true);
    sky_node->attach(m_viewer, 0);
    sky_node->setSunVisible(true);
    sky_node->setStarsVisible(true);    
    sky_node->setEphemeris(ephemeris);
    bool re = sky_node->getStarsVisible();

    return sky_node;
}

**Comment by gwaldron** [comment]
Please share your platform, version, and a complete .cpp that will reproduce the issue. Thanks.
